README.md

```diff
@@ -0,0 +1,143 @@
+# NixOS on the HoneyComb / ClearFog LX2k
+
+This repo contains minimal viable Nix expressions for NixOS installer images and UEFI
+firmwares for the following machines:
+
+* [HoneyComb LX2K][lx2k-store]
+
+# Prerequisites
+
+Before starting, you should ensure you have:
+
+* Any additional required hardware is properly installed.
+  The default memory speed is 3200mhz. If your memory is slower, you will need to specify a different speed.
```

Comment:
Maybe we should default to 2400MHz for maximum compatibility.

Reply:
How about no default, so the arg has to be passed?

Reply:
I think 2400MHz is a good default since it'll work with any RAM, but I don't feel strongly about this.